code/Test_definitions/DeviceIdentifier_retrieve_identifier.feature

```diff
@@ -0,0 +1,148 @@
+@DeviceIdentifier_retrieve_identifier
+Feature: Camara Device Identifer API retrieve identifier
+
+# Input to be provided by the implementation to the tests
+# References to OAS spec schemas refer to schemas specified in code/API_definitions/CAMARA Mobile Device Identifier API.yaml
+# Implementation indications:
+# * api_root: API root of the server URL
+#
+# Testing assets:
+# * a mobile device with imei IMEI1 and phone number PHONENUMBER1
+# * a mobile device with imei IMEISV2  and phone number PHONENUMBER2
+# * a mobile device with DeviceIpv4Addr value IPV4ADDRESS
```

Comment:
`DeviceIpv4Addr` is a complex object, so you need to pick one of the valid options. Most likely is:
`a mobile device with imei IMEI2, public IPv4 address PUBLICIPV4ADDRESS and public port PUBLICPORT`